### ISSUE TITLE
Stick breaking bijector

### ISSUE THREAD

**Issue by dustinvtran** [tensorflow/probability#6]
Motivated by conversation in https://github.com/probtorch/pytorch/issues/112

**Comment by junpenglao** [comment]
Is bijector going to be merged into tensorflow/probability? or contribution should be submitted directly to tensorflow?

**Comment by srvasude** [comment]
Hi Junpeng,
  As of this moment bijectors and distributions are living in core tensorflow. However, in the future I think we would like to move these over here. Contributions can continue to go to tensorflow, and hopefully in the near future they will all be moved here.

 I also believe tfp.distributions should work (vs. tf.contrib.distributions), so contributing to core tensorflow still allows you to use the tensorflow_probability namespace :).

**Comment by dustinvtran** [comment]
@srvasude: Are you working on this? Otherwise, do you think this is a "good first issue" to label for external contributors?

**Comment by srvasude** [comment]
I'm not actively working on this (but might get to at some point). This could be a good first issue (although there are nuances to stick breaking. The ordered bijector might be a better "good first issue").

**Comment by dustinvtran** [comment]
Got it. I'll leave you as assigned for this one and label the ordered bijector as "good first issue".

**Comment by twiecki** [comment]
Any progress on this? This is a fairly important bijector.

**Comment by srvasude** [comment]
I recently wrote an internal PR for this (and just need to respond to some comments).
